perf-testing/test_local_only.rb

```diff
@@ -0,0 +1,267 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.unshift(File.expand_path('../gems/aws-sdk-core/lib', __dir__))
+$LOAD_PATH.unshift(File.expand_path('../gems/aws-sdk-echo/lib', __dir__))
+
+require 'aws-sdk-core'
+require 'aws-sdk-echo'
+require 'aws-sdk-core/plugins/protocols/json_rpc'
+require 'aws-sdk-core/plugins/protocols/rpc_v2'
+require_relative 'Stats'
+include Stats
+
+ITERATIONS = ARGV.first.to_i
+WARMUP = 10
+ASCII = "!\"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~".chars
+INT_MIN = -2_147_483_648
+INT_MAX = 2_147_483_647
+LONG_MIN = -9_223_372_036_854_775_808
+LONG_MAX = 9_223_372_036_854_775_807
+FLOAT_MIN = -2_147_483_648.0
+FLOAT_MAX = 2_147_483_647.0
+DOUBLE_MIN = -9_223_372_036_854_775_808.0
+DOUBLE_MAX = 9_223_372_036_854_775_807.0
+RUN_START_TIMESTAMP = Time.now
```

Comment:
Ditto here - this value changes each run, so I'd potentially not case it as a constant.